src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -36,6 +45,10 @@ public void contextInitialized(ServletContextEvent sce) {
       List<Event> events = PersistentStorageAgent.getInstance().loadEvents();
       EventStore.getInstance().setEvents(events);
 
+      NemoBot nemoBot = new NemoBot();
+      botController.registerBot("@NemoBot", nemoBot);
```

Comment:
We can use the bot name here instead of hardcoding it twice:

`botController.registerBot(nemoBot.getName(), nemoBot);`

We can add the `@` when looking up bots with the mention key.